tools/pipelines/test-real-service.yml

```diff
@@ -216,4 +216,3 @@ stages:
                   node "$BUILD_SOURCES_DIRECTORY/scripts/get-test-pass-rate.mjs"
                   echo "Files created. Now running telemetry-generator"
                   node --require @ff-internal/aria-logger bin/run --handlerModule "$(toolAbsolutePath)/dist/handlers/testPassRate.js" --dir "$BASE_OUTPUT_FOLDER/stageFiles"
-                
```

Comment:
Unrelated to this PR: I thought our auto-formatting rules took care of this but I've seen changes around end-of-file newlines recently. Is it not enforced one way or the other by our repo setup?

Reply:
Reformatting YAML is generally dangerous, at least for CI workflows.

https://github.com/microsoft/FluidFramework/blob/96d9e2ac87de0cb3614ecb1ac759a3d7b9a045e4/.prettierignore#L29-L30

Reply:
And AFAIK prettier is an "all or nothing" formatter, so there's not an easy way to apply some "rules" but not others, at least with prettier alone.